.travis.yml

```diff
@@ -1,50 +0,0 @@
-os: linux
-dist: bionic
-env:
-# See, https://docs.travis-ci.com/user/speeding-up-the-build/
-# We need a balanced distribution of the tests
-# Once we add and remove tests, this distribution may become unbalanced.
-# Feel free to move tests around to make the running time of the jobs
-# as close as possible.
-  - TESTS=test_[a-e]*
-  - TESTS=test_[f-h]*
-  - TESTS=test_[i-r,t-z]*
-# test_simple_commands.py is slow due to parametrization.
-  - TESTS=test_[s]*
-  - TESTS=webextension
-git:
-  depth: 3
-before_install:
-  - "export DISPLAY=:99.0"
-  - "export BOTO_CONFIG=/dev/null" # https://github.com/travis-ci/travis-ci/issues/7940
-install:
-  - ./scripts/install-miniconda.sh
-  - source "$HOME/miniconda/etc/profile.d/conda.sh"
-  - ./install.sh
-  - conda activate openwpm
-before_script:
-  - if [[ "$TESTS" != "webextension" ]]; then
-      pre-commit run --all;
-    fi
-script:
-  - ./scripts/travis.sh
-after_success:
-  - if [[ "$TESTS" == "webextension" ]]; then
-      npm run cov:check;
-      npm run cov:send;
-    fi
```

Comment:
Should we port this to the Github Actions workflow?

Reply:
Done!